### DISCUSSION TITLE
UEFI BIOS下Windows系统的安装

### DISCUSSION THREAD

**Discussion by giscus** [w1zd/w1zd.github.io#57] [Announcements]
# UEFI BIOS下Windows系统的安装

Enjoy life && Enjoy Coding

https://wizd.dev/UEFI-BIOS%E4%B8%8BWindows%E7%B3%BB%E7%BB%9F%E7%9A%84%E5%AE%89%E8%A3%85/

**Comment by w1zd** [comment]
多年后看这个，有点傻。。。
